src/components/layout/Feedback/Feedback.module.scss

```diff
@@ -0,0 +1,105 @@
+@import "../../../styles/settings.scss";
+
+.root {
+  margin-bottom: 30px;
+  .panelBar {
+    margin-bottom: 30px;
+    position: relative;
+
+    :global(.row) > * {
+      border-bottom: 4px solid $global-border-color;
+    }
+
+    .heading {
+      position: relative;
+      width: 100%;
+
+      h3 {
+        color: $primary;
+        text-transform: uppercase;
+        font-size: 20px;
+        line-height: 38px;
+        margin: 0;
+        letter-spacing: 1px;
+      }
+
+      &::before {
+        content: "";
+        position: absolute;
+        bottom: -4px;
+        left: 0;
+        width: 45px;
+        border-bottom: 4px solid $primary;
+      }
+    }
+
+    .dots {
+      position: absolute;
+      right: 0;
+      top: 0;
+
+      ul {
+        margin: 0;
+        padding: 0 0 0 1rem;
+        list-style: none;
+        background-color: $color-light-primary;
+        transform: translateY(15px);
+
+        li {
+          display: inline-block;
+          margin-left: 0.5rem;
+
+          a {
+            display: block;
+            width: 13px;
+            height: 13px;
+            border-radius: 6px;
+            background-color: $color-light-secondary;
+            font-size: 0;
+
+            &.active,
+            &:hover {
+              background-color: $primary;
+            }
+          }
+        }
+      }
+    }
+  }
+  .imageBox {
+    overflow: hidden;
+    height: 75px;
+  }
+  
+  img {
+    width: 50px;
+    height: 100%;
+    text-align: center;
+    object-fit: cover;
+    position: absolute;

```

Comment:
Bez tego absolute, nie jest potrzebny

Reply:
bez absolute ucieka mi  zdjecie do lewej jest jakies inne na to rozwiazanie ?